daemon/archive.go

```diff
@@ -257,12 +241,6 @@ func (container *Container) ExtractToDir(path string, noOverwriteDirNonDir bool,
 		return ErrExtractPointNotDirectory
 	}
 
-	baseRel, err := filepath.Rel(container.basefs, resolvedPath)
-	if err != nil {
-		return err
-	}
-	absPath = filepath.Join("/", baseRel)
-
```

Comment:
This does not seem right. If the path is `/myvolume/symlinktoroot/` it would be matched to the volume and writing would go to the readonly rootfs.